Workflow.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:30:17 2016
+
+@author: mina
+"""
+
+import yaml
+
+class Workflow:
+    def __init__(adress,self):      
+      with open(adress, 'r') as f:
+          doc = yaml.load(f)
```

Comment:
At this point, we could store the config dictionary as something like `self.config = yaml.load(f)`. The yaml file may not be correctly formatted so we could wrap this section in a separate try/except and let the user know that the yaml file was not formatted properly.